app/views/digitalpack/country.scala.html

```diff
@@ -3,36 +3,35 @@
 @main("Select your country | Digital | Subscriptions | The Guardian", bodyClasses = List("is-wide")) {
     <main class="page-container gs-container">
 
-    <section class="checkout-container">
+        <section class="checkout-container">
```

Comment:
if you suffix the PR files/ URL with ?w=1 whitespace only line changes get ignored, which is handy for situations like this